doc/07_taxonomic_discovery_with_sourmash.md

```diff
@@ -59,60 +59,152 @@ A signature is a compressed representation of the k-mers in the sequence.
 Using this data structure instead of the reads makes sourmash much faster.
 
 ```
-sourmash sketch -o SRR1976948.sig --name SRR1976948 -p scaled=2000,k=21,k=31,k=51,abund SRR1976948_*fastq.gz
+sourmash sketch dna -p scaled=1000,k=21,k=31,k=51,abund SRR1976948_*fastq.gz --name SRR1976948 -o SRR1976948.sig
 ```
 
-The outputs file, `SRR1976948.sig` holds a representative subset of k-mers from our original sample, as well as their abundance information. 
+> - `sourmash sketch dna` creates a signature file
+> - `-p scaled=1000,k=21,k=31,k=51,abund` are the creation parameters of the signature file 
+> - `SRR1976948_*fastq.gz --name SRR1976948` creates a single signature from multiple files
+> - `-o SRR1976948.sig` outputs the signature to the designated name `SRR1976948.sig`
+
+The outputs file, `SRR1976948.sig` holds a representative subset of k-mers from our original sample, as well as their abundance information.
 The k-mers are "hashed", or transformed, into numbers to make selecting, storing, and looking up the k-mers more efficient.
+In addition, these techniques remove the necessity to trim the raw fasta files before processing as the likelihood of finding a matching erroneous hash is statistically **extremely** unlikely.
+
+To quickly peak at a signature file, database, or collection created by sourmash. Use:
+
+```
+sourmash sig summarize SRR1976948.sig
+```
+
+This will output:
+
+```
+== This is sourmash version 4.8.5. ==
+== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==
+
+** loading from 'SRR1976948.sig'
+path filetype: MultiIndex
+location: SRR1976948.sig
+is database? no
+has manifest? yes
+num signatures: 3
+** examining manifest...
+total hashes: 3007375
+summary of sketches:
+   1 sketches with DNA, k=21, scaled=1000, abund      838571 total hashes
+   1 sketches with DNA, k=31, scaled=1000, abund      1006183 total hashes
+   1 sketches with DNA, k=51, scaled=1000, abund      1162621 total hashes
+```
 
 ## Sourmash gather
 
 Sourmash provides several methods for estimating the composition of known sequences in a metagenome. `sourmash gather` is the primary technique ([ref](https://www.biorxiv.org/content/10.1101/2022.01.11.475838)) -
 it gives strain-level specificity to matches in its output -- e.g. all strains that match any sequences (above a threshold) in your metagenome will be reported, along with the percent of each strain that matches. 
 This is useful both to estimate the amount of your metagenome that is known, and to estimate the closest strain relative to the thing that is in your metagenome. 
 
-Download and unzip the sourmash database:
+Download a pre-built [sourmash database](https://sourmash.readthedocs.io/en/latest/databases.html#gtdb-r08-rs214-genomic-representatives-85k):
 
 ```
-curl -L https://osf.io/4f8n3/download -o genbank-k31.lca.json.gz
-gunzip genbank-k31.lca.json.gz
+curl -JLO https://farm.cse.ucdavis.edu/~ctbrown/sourmash-db/gtdb-rs214/gtdb-rs214-reps.k31.lca.json.gz
```

Comment:
assuming we're running this on farm, you can just point people at `/group/ctbrowngrp/sourmash-db/gtdb-rs214`!

Reply:
also, with `gather`, no need to use the LCA database (which is bigger memory, tho maybe faster). could just as easily use the regular zip.

Reply:
>  assuming we're running this on farm, you can just point people at /group/ctbrowngrp/sourmash-db/gtdb-rs214!

Would that be a good time to introduce `ln -s` as well?

> also, with gather, no need to use the LCA database (which is bigger memory, tho maybe faster). could just as easily use the regular zip.

I did add a ***Bonus*** goal at the end of the document to use the standard zip database, I think I should switch them around. Introduce the zip db in the document and add a bonus about using other database types...?

Reply:
> > assuming we're running this on farm, you can just point people at /group/ctbrowngrp/sourmash-db/gtdb-rs214!
> 
> Would that be a good time to introduce `ln -s` as well?

sure!

> > also, with gather, no need to use the LCA database (which is bigger memory, tho maybe faster). could just as easily use the regular zip.
> 
> I did add a _**Bonus**_ goal at the end of the document to use the standard zip database, I think I should switch them around. Introduce the zip db in the document and add a bonus about using other database types...?

zip better. Not sure why other database types needed here, but sure, in the bonus section... :)